llvm/docs/DirectX/DXILResources.rst

```diff
@@ -197,23 +200,23 @@ Examples:
 .. code-block:: llvm
 
    ; RWBuffer<float4> Buf : register(u5, space3)
-   %buf = call target("dx.TypedBuffer", float, 1, 0)
+   %buf = call target("dx.TypedBuffer", <4 x float>, 1, 0, 0)
                @llvm.dx.handle.fromBinding.tdx.TypedBuffer_f32_1_0(
                    i32 3, i32 5, i32 1, i32 0, i1 false)
 
-   ; RWBuffer<uint> Buf : register(u7, space2)
-   %buf = call target("dx.TypedBuffer", i32, 1, 0)
+   ; RWBuffer<int> Buf : register(u7, space2)
+   %buf = call target("dx.TypedBuffer", i32, 1, 0, 1)
```

Comment:
You say in this doc that `IsSigned` is an extra parameter, but that param is only present for `TypedBuffer`. Yet, I see a new parameter here.
Maybe you meant to write somewhere that RWBuffers are represented as TypedBuffers? Should that be made explicit? (Like you specified with ByteAddressBuffers above).
Otherwise, shouldn't we specify that RWBuffers also get an extra parameter, `IsSigned`?

Reply:
I thought it was fairly explicit already, but I've added some wording to be clearer about that.